test/integration/wyatt_test.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+class WyattTest < ActionDispatch::IntegrationTest
+  test "shows specific message when visiting '/wyatt'" do
+    get "/wyatt"
+    assert_response :success
+
+    assert_select "p", "Hi James!"
```

Comment:
Here we are making another assertion for the test, this assertion says, "find a `<p>` tag in the response and make sure it contains the text 'Hi James!'."